### ISSUE TITLE
Have an option to auto close update page after update

### ISSUE THREAD

**Issue by peter-kehl** [huhu/rust-search-extension#110]
The update popup can be annoying. Please add an option not to show it (but still to update).

**Comment by Folyd** [comment]
That's weird. Auto-opening the update page is disabled by default. You can disable the auto-update in the popup pages if it has been open accidentally.

![image](https://user-images.githubusercontent.com/3369694/112652042-f231a100-8e87-11eb-8fcd-fcb9c5d044a9.png)

**Comment by peter-kehl** [comment]
This request is not about enabling/disabling auto-update by default.

Instead, I'm asking for a new option/checkbox: Do auto-update, but don't show any popup.

**Comment by Folyd** [comment]
> This request is not about enabling/disabling auto-update by default.
> 
> Instead, I'm asking for a new option/checkbox: Do auto-update, but don't show any popup.

I'm afraid we can't achieve silence auto-update due to the API limitation of the browser. See this issue #97.

**Comment by peter-kehl** [comment]
OK. Thank you.

**Comment by aminya** [comment]
Yeah, this extension randomly opens a new page every now and then, although I did not ask for it.

**Comment by Folyd** [comment]
Hi @aminya. Thanks for reporting. Could you please check the `Enable auto update` is turned off?

![image](https://user-images.githubusercontent.com/3369694/133921207-be1bae0a-383f-45d6-836d-a5df35405e85.png)

**Comment by aminya** [comment]
It was enabled for me. Why is it necessary to open a new tab for auto updating? There should be a way to do it without this

**Comment by Folyd** [comment]
There has a declaration about the intent of auto-update on the [update page](https://rust.extension.sh/update). See https://github.com/huhu/rust-search-extension/issues/97.
![image](https://user-images.githubusercontent.com/3369694/133924592-6a5660d2-9f90-4c71-9691-3f392e9ab79b.png)

**Comment by nihaals** [comment]
Maybe this should be renamed to auto close update page after update?

**Comment by nihaals** [comment]
Also it would be nice to have this be a separate setting toggle as some users might be confused/concerned by a tab quickly opening and closing, similar to Windows and random `cmd.exe` windows.

**Comment by nihaals** [comment]
Was this implemented or is this closed as incomplete?

**Comment by peter-kehl** [comment]
https://addons.mozilla.org/en-US/firefox/addon/rust-search-extension/versions and https://github.com/huhu/rust-search-extension/releases show a new version. But https://github.com/huhu/rust-search-extension/commits/master doesn't show any commit message mentioning auto closing the update tab/window. Yet to see.
